### DISCUSSION TITLE
jdownloader in container manager, Synology

### DISCUSSION THREAD

**Discussion by Danielnope** [jlesage/docker-jdownloader-2#173] [Q&A]
I created the jdownloader in the Container Manager on my Synology under DSM 7.2. The container  starts without any problems. When I look at the GUI, "IP address":5800 I get the message no connection to the update service, and the possibility to set other connection options, for example direct, host and so on.

**Comment by jlesage** [comment]
Make sure in the container's configuration that port 5800 is really mapped to 5800.  If you didn't set the mapping yourself, Container Manager will use a random port (that you need to use to connect).

**Comment by lovehumans** [comment]
@Danielnope your issue is that the container is running in the "bridge" network instead of the "host" network, so it does not have access to the internet.
